File: app/src/main/java/com/dk/tengerms/views/fragments/Checkout.java
package com.dk.tengerms.views.fragments;

import android.os.Bundle;
import android.os.Handler;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;
import android.widget.Button;

import com.dk.tengerms.R;
import com.dk.tengerms.controllers.Dashboard;

import androidx.annotation.NonNull;
import androidx.annotation.Nullable;
import androidx.fragment.app.Fragment;


public class Checkout extends Fragment implements View.OnClickListener {
    Button bt_login;

    @Override
    public void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
    }

    @Override
    public View onCreateView(LayoutInflater inflater, ViewGroup container,
                             Bundle savedInstanceState) {
        // Inflate the layout for this fragment
        return inflater.inflate(R.layout.fragment_checkout, container, false);
    }

    @Override
    public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceState) {
        super.onViewCreated(view, savedInstanceState);
        init(view);
    }

    public void init(View view) {
        bt_login = view.findViewById(R.id.bt_login);
        bt_login.setOnClickListener(this);
    }

    @Override
    public void onClick(View v) {
        switch (v.getId()) {
            case R.id.bt_login:
                login();
                break;

        }
    }

    private void login() {
        ((Dashboard)getActivity()).globalClass.showDialog(getActivity());
        final Handler handler = new Handler();
        handler.postDelayed(new Runnable() {
            @Override
            public void run() {
                ((Dashboard)getActivity()).globalClass.hideLoader();
//                NavHostFragment.findNavController(Checkout.this).navigate(R.id.login_to_dashboard);
            }
        }, 1500);
    }
}

File: app/src/main/java/com/paxees/tcc/views/fragments/drawerMenusFragments/Checkout.kt
package com.paxees.tcc.views.fragments.drawerMenusFragments

import android.os.Bundle
import android.os.Handler
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import android.widget.Button
import androidx.fragment.app.Fragment
import com.paxees.tcc.R
import com.paxees.tcc.controllers.Dashboard

class Checkout : Fragment(), View.OnClickListener {
    var bt_login: Button? = null
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
    }

    override fun onCreateView(
        inflater: LayoutInflater, container: ViewGroup?,
        savedInstanceState: Bundle?
    ): View? {
        // Inflate the layout for this fragment
        return inflater.inflate(R.layout.fragment_checkout, container, false)
    }

    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
        super.onViewCreated(view, savedInstanceState)
        init(view)
    }

    fun init(view: View) {
        bt_login = view.findViewById(R.id.bt_login)
        bt_login!!.setOnClickListener(this)
    }

    override fun onClick(v: View) {
        when (v.id) {
            R.id.bt_login -> login()
        }
    }

    private fun login() {
        (activity as Dashboard?)!!.globalClass.showDialog(activity)
        val handler = Handler()
        handler.postDelayed({
            (activity as Dashboard?)!!.globalClass.hideLoader()
            //                NavHostFragment.findNavController(Checkout.this).navigate(R.id.login_to_dashboard);
        }, 1500)
    }
}